[SetGridField] Support for setting emtpy fields in XSIAM (#30032)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_12_30.md

#### Scripts

##### SetGridField
- Updated the Docker image to: *demisto/pandas:1.0.0.76121*.

> - Fixed an issue where setting empty grid fields was failing in XSIAM.

Reviewer: ```suggestion
- Fixed an issue where the automation failed when setting empty grid fields on XSIAM.
```
